src/object/srv_obj_remote.c

```diff
@@ -124,21 +124,9 @@ ds_obj_remote_update(struct dtx_leader_handle *dlh, void *data, int idx,
 	orw_parent = crt_req_get(parent_req);
 	orw = crt_req_get(req);
 	*orw = *orw_parent;
-	if (split_req != NULL) {
-		tgt_idx = shard_tgt->st_shard_id;
-		tgt_oiod = obj_ec_tgt_oiod_get(split_req->osr_tgt_oiods,
-					       dlh->dlh_normal_sub_cnt + dlh->dlh_delay_sub_cnt + 1,
-					       tgt_idx - obj_exec_arg->start);
-		D_ASSERT(tgt_oiod != NULL);
-		orw->orw_iod_array.oia_oiods = tgt_oiod->oto_oiods;
-		orw->orw_iod_array.oia_oiod_nr = orw->orw_iod_array.oia_iod_nr;
-		orw->orw_iod_array.oia_offs = tgt_oiod->oto_offs;
-	}
```

Comment:
does it mean here the leader forward the same iods/csums to tgt servers?
this looks worse than original code that only forward tgt server needed information that could be much shorter size, and this change could cause RPC inter-operability issue, or I may misunderstood it?

Reply:
See the comments above, though we can discuss further.